src/NClient.Tools/NClient.DotNetTool/CommandLineOptions.cs

```diff
@@ -0,0 +1,28 @@
+using CommandLine;
+
+namespace NClient.DotNetTool
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    public class CommandLineOptions
+    {
+        public CommandLineOptions(string spec, string outputPath, string @namespace, string facadeName)
+        {
+            Spec = spec;
+            OutputPath = outputPath;
+            Namespace = @namespace;
+            FacadeName = facadeName;
+        }
+
+        [Option(shortName: 's', longName: "spec", Required = true, HelpText = "The OpenAPI spec file to use. Paths are relative to the project directory.")]
+        public string Spec { get; }
```

Comment:
I failed to generate facades for this specification source: https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v2.0/json/uber.json
The tool did not generate interfaces, but there were no errors in the logs

Reply:
I've added some "ensure" in [commit](https://github.com/nclient/NClient/pull/360/commits/ad5f83158c251e31da2fb9496e722cc2a815ad70)